scheduler/pkg/kafka/pipeline/grpcserver.go

```diff
@@ -88,6 +88,7 @@ func (g *GatewayGrpcServer) Start() error {
 		opts = append(opts, grpc.Creds(g.tlsOptions.Cert.CreateServerTransportCredentials()))
 	}
 	opts = append(opts, grpc.MaxConcurrentStreams(maxConcurrentStreams))
+	opts = append(opts, grpc.MaxRecvMsgSize(util.GrpcMaxMsgSizeBytes))
```

Comment:
Same point here as in other server components about setting the max send size to match this.  A response from Kafka might be larger than the input (e.g. in the case of something generative, an up-scaler, etc.).